biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -387,16 +391,29 @@ subroutine LoggingMortality_frac( currentSite, bc_in, pft_i, dbh, canopy_layer,
          endif
 
       else
-         call GetInitLanduseHarvestRate(bc_in, currentSite%min_allowed_landuse_fraction, &
-              harvest_rate, currentSite%landuse_vector_gt_min)
-         lmort_direct     = 0.0_r8
-         lmort_collateral = 0.0_r8
-         lmort_infra      = 0.0_r8
-         l_degrad         = 0.0_r8
-         if(prt_params%woody(pft_i) == itrue)then
-            lmort_direct     = harvest_rate
-         else if (canopy_layer .eq. 1) then
-            l_degrad         = harvest_rate
+         ! the logic below is mainly to prevent conversion of bare ground land; everything else should be primary at this point.
+         if ( patch_land_use_label .eq. primaryland ) then
+            call GetInitLanduseHarvestRate(bc_in, currentSite%min_allowed_landuse_fraction, &
+                 harvest_rate, currentSite%landuse_vector_gt_min)
+            lmort_direct     = 0.0_r8
+            lmort_collateral = 0.0_r8
+            lmort_infra      = 0.0_r8
+            l_degrad         = 0.0_r8
+            if(prt_params%woody(pft_i) == itrue)then
+               lmort_direct     = harvest_rate
+            else if (canopy_layer .eq. 1) then
+               l_degrad         = harvest_rate
+            endif
+         else if ( patch_land_use_label .eq. nocomp_bareground_land ) then
+            lmort_direct     = 0.0_r8
+            lmort_collateral = 0.0_r8
+            lmort_infra      = 0.0_r8
+            l_degrad         = 0.0_r8
```

Comment:
Can be simplified by putting `lmort_direct = 0.0_r8` etc. in front of the `if` block.